pom.xml

```
@@ -559,8 +536,8 @@
                 <configuration>
                     <skipCheckoutIfExists>false</skipCheckoutIfExists>
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-                    <scmVersionType>tag</scmVersionType>
-                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-3137-remove-xpayvpos-refs</scmVersion>
```

TODO: to be restored after dependent pr merge
```suggestion
                    <scmVersionType>tag</scmVersionType>
                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
```